solr/solrj/src/java/org/apache/solr/client/solrj/io/eval/GaussFitEvaluator.java

```diff
@@ -81,7 +80,7 @@ public Object doWork(Object... objects) throws IOException{
 
     double[] coef = curveFitter.fit(pointList);
     Gaussian gaussian = new Gaussian(coef[0], coef[1], coef[2]);
-    List list = new ArrayList();
+    List<Double> list = new ArrayList<>();
```

Comment:
I'm not sure I understand the suggestion - `VectorFunction` takes a List<? extends Number> specifically so that we can have the flexibility to use `List<Double>` or `List<Integer>` or whatever in other places.